cities/modeling/modeling_utils.py

```diff
@@ -91,49 +91,17 @@ def prep_wide_data_for_inference(
     assert f_covariates_joint["GeoFIPS"].equals(intervention["GeoFIPS"])
 
     # extract data for which intervention and outcome overlap
-    year_min = max(
-        intervention.columns[2:].astype(int).min(),
-        outcome.columns[2:].astype(int).min(),
-    )
-
-    year_max = min(
-        intervention.columns[2:].astype(int).max(),
-        outcome.columns[2:].astype(int).max(),
-    )
-
-    assert all(intervention["GeoFIPS"] == outcome["GeoFIPS"])
-
-    outcome_years_to_keep = [
-        year
-        for year in outcome.columns[2:]
-        if year_min <= int(year) <= year_max + forward_shift
-    ]
-
-    outcome_years_to_keep = [
-        year for year in outcome_years_to_keep if year in intervention.columns[2:]
-    ]
-
```

Comment:
The line 106 to line 114 works the same as 
`
outcome_years_to_keep = [year for year in outcome.columns[2:] if year in intervention.columns[2:]
`
It seems that the intent was to write 
`
outcome_years_to_keep = [year for year in outcome.columns[2:] if year - forwardshift in intervention.columns[2:]
`

Reply:
Alright, I think the issue was dropping a variable that was needed elsewhere.